package.json

```diff
@@ -6,17 +6,20 @@
     "start": "nodemon ./src/bin/www"
   },
   "dependencies": {
-    "cookie-parser": "~1.4.4",
-    "debug": "~2.6.9",
-    "express": "~4.16.1",
-    "http-errors": "~1.6.3",
-    "morgan": "~1.9.1",
-    "pg": "^8.4.2",
-    "pug": "2.0.0-beta11",
-    "sequelize": "^6.3.5"
+    "cookie-parser": "1.4.6",
```

Comment:
The [`~` and `^` versioning](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) is key to a few of these dependencies -- we want them to continuously update to the latest patches, rather than being hardcoded

Reply:
makes sense will make the required corrections